### PR TITLE
Fetching complete data from MongoDB, dropdown feature, CSS changes, title changes, fetch url changed.

### PR DESCRIPTION
**Pull request by ShreyAswal**
- Hosted all categories on MongoDB database 
- Fetching all data in landing page from database
- Made a dropdown feature for mobile users
- Updated post request URL
- Updated CSS 
- Updated Titles/ Categories
- Deleted all necessary code

### CODE REVIEW

File: EinvitationMaker/src/components/CategoriesData.js [hunk 1]
```diff
@@ -2,9 +2,28 @@
 import btemp1 from "./Templates/btemp1SS.png";
 import wedAnniv1 from "./Templates/Wed1SS.png";
 import wtemp1 from "./Templates/wedAnnivSS.png";
+// import btemp1 from "./Templates/Btemp1"
```
Comment (PatelNeelMahesh) [inline EinvitationMaker/src/components/CategoriesData.js]: Delete the commented code, from all files. Later if you need you can refer previous commits.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
After removing the commented code you can merge pull request.

File: EinvitationMaker/src/components/CategoriesData.js (hunk 1 above)
Comment (ShreyAswal) [reply EinvitationMaker/src/components/CategoriesData.js]: I have already made more changes so after completing the next feature, will delete the commented lines and do another PR.

Comment (ShreyAswal) [reply EinvitationMaker/src/components/CategoriesData.js]: Done.

**Review by PatelNeelMahesh — APPROVED** [review]

**Pull request merged**